Answers.md

```diff
@@ -2,10 +2,18 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+React attempts to manuplate the DOM faster. A virtual dom is rendered through a tree of javascript. React compares the virtual dom with the previous to figure out what needs to change. Then react applies those changes at a quicker rate. 
+
 1.  What does it mean to _think_ in react?
 
+To think in react, you break the UI into component hierarcy. A static version is built, followed by identifying the minimal representation of UI state. Then state is identified a place to be kept and data flow is added. 
+
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
 
+A breif description of the difference betwee a class/stateful component and a functional/presentational component is that a class/stateful component has state.
+

```

Comment:
True, also be sure to know that functional components essentially have one job, and that is to render data that is passed to them to the DOM.